cmd/mount_test.go

```diff
@@ -427,6 +427,26 @@ func (suite *mountTestSuite) TestFuseOptions() {
 	}
 }
 
+func (suite *mountTestSuite) TestUpdateCliParams() {
+	defer suite.cleanupTest()
+
+	cliParams := []string{"blobfuse2", "mount", "~/mntdir/", "--foreground=false"}
```

Comment:
We shall  test for overwrite as well, like tmp-path is there in cliParams already then it shall not be added again or overwritten.

Reply:
Already checking this below.
```
updateCliParams(&cliParams, "tmp-path", "tmpPath2")
updateCliParams(&cliParams, "container-name", "testCnt2")
suite.assert.Equal(len(cliParams), 6)
suite.assert.Equal(cliParams[4], "--tmp-path=tmpPath2")
suite.assert.Equal(cliParams[5], "--container-name=testCnt2")
```